core/domain/feedback_services.py

```diff
@@ -366,6 +366,30 @@ def _get_thread_from_model(thread_model):
         thread_model.created_on, thread_model.last_updated)
 
 
+def get_new_messages_for_exp_id(exp_id):
+    """Fetch the feedback threads for the given exploration, and use those to
+    finally get the new (open) feedback messages.
```

Comment:
I traced this code path, but I don't see where "open" comes into it. Where are you filtering on 'open'?

Also, I wonder if we could standardize the naming a bit. We have get_new_messages_for_exp_id() which calls get_latest_updated_threads(). Maybe they should be get_most_recent_messages() and get_most_recently_updated_threads()?